rholang/src/test/scala/coop/rchain/rholang/PeekSpec.scala

```diff
@@ -77,4 +77,37 @@ class PeekSpec extends FlatSpec with Matchers {
       }
       .runSyncUnsafe(2.seconds)
   }
+
+  it should "clear all peeks when inserting a persistent send" in {
+    mkRuntime[Task](tmpPrefix)
+      .use { runtime =>
+        implicit val c = runtime.cost
+        for {
+          _   <- evaluate[Task](runtime, """for (_ <<- @0) { @1!(0) }""")
+          _   <- evaluate[Task](runtime, """for (_ <<- @0) { @1!(0) }""")
+          _   <- evaluate[Task](runtime, """@0!!(0)""")
```

Comment:
May I ask a test like this one, but with `@0!(0)` too? :)
("clear all peeks when inserting a(ny) send")

Reply:
Can do